src/OpenSpaceToolkit/Core/Containers/Array.tpp

```diff
@@ -23,6 +23,12 @@ namespace ctnr
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+                                template <class T>
```

Comment:
Added functions not used yet in effect but could be useful for proper automatic implicit casting techniques used by pybind11 in the future

Reply:
Removed for now